NewForce-Cohort-4/nutshell-frugal-badgers
# News Articles Widget

# Description
Implemented ticket #2: "As a user, I should be able to list interesting news articles on my dashboard"

# Testing Instructions
With local json server running and VS code hosting, you should be able to log in as any user and see a button to add news articles under the News section of the dashboard. Once clicked, a save form is created to save a title, URL, and summary. After hitting save, you should see all articles that user has saved displayed in the dashboard widget. Articles are shown by most recently saved at the top. Each article has a delete button that will remove that article from the user's account.

# Checklist:
